app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,77 @@
+class TweetsController < ApplicationController
+   
+   get '/tweets' do 
+    if is_logged_in?
+    
+     erb :'/tweets/tweets'
+    else
+    redirect '/login'
+    end
+  end
+
+  get '/tweets/new' do
+   if is_logged_in?
+    erb :'/tweets/create_tweet'
+   else 
+    redirect '/login'
+    end
+  end
+
+post '/tweets/new' do
+   if !params[:content].empty? 
+      @tweet = current_user.tweets.create(content: params[:content])
+      redirect  to "/tweets/#{@tweet.id}"
+   else
+     redirect "/tweets/new"
+   end
+  end
+  
+  get '/tweets/:id' do
+    if is_logged_in?
+    @tweet = Tweet.find_by_id(params[:id])
+    erb :'/tweets/show_tweet'
+    else
+    redirect to '/login'
+    end
+  end
+  
+  get '/tweets/:id/edit' do
+    if is_logged_in?
+      @tweet = Tweet.find(params[:id])
+      if @tweet.user_id == session[:user_id]
```

Comment:
@johnsontaiwo you have a current_user method that you should be using here instead of `session[:user_id]`. `current_user.id` would be better.